util/cli/common_options.go

```diff
@@ -0,0 +1,20 @@
+package cli
+
+import (
+	"github.com/src-d/lookout/util/grpchelper"
+)
+
+type initializer interface {
```

Comment:
Again, most probably a nitpick, but would `initializable` better represent it's purpose?

Then the client might look something like
```go
if v, ok := command.(initializable); ok {
	v.initialize(app.Name)
}
```

Reply:
https://golang.org/doc/effective_go.html#interface-names